app/queries/developers/query_item_builder.rb

```diff
@@ -0,0 +1,35 @@
+class Developers::QueryItemBuilder
+
+  attr_reader :query_items, :options
+
+  QUERY_ITEM_OPTIONS = %i[sort countries]
+
+  def initialize(options = {})
+    @options = options
+    @query_items = []
+  end
+
+  def build_query_items
+    add_sorting_query
+    add_countries_filter_query
```

Comment:
I think that these two should be passed in as parameters to `initialize`. That way we can pass in any combination we want - and future queries only require changes to the new query item and not this class.